cmd/ocsp-updater/main.go

```diff
@@ -23,20 +28,74 @@ import (
 	"github.com/letsencrypt/boulder/sa"
 )
 
-// FatalError indicates the updater should stop execution
-type FatalError string
-
-func (e FatalError) Error() string { return string(e) }
-
 // OCSPUpdater contains the useful objects for the Updater
 type OCSPUpdater struct {
 	stats statsd.Statter
 	log   *blog.AuditLogger
-	cac   rpc.CertificateAuthorityClient
+	clk   clock.Clock
+
 	dbMap *gorp.DbMap
+
+	rpcMu *sync.RWMutex
```

Comment:
Well, this was the second plan I came up with, after I first recommended that @rolandshoemaker just tear out the reconnect code for now and exit on connection failure.

What are your thoughts on the fake AmqpRPCServer notion?